extensions/mssql/src/objectManagement/commands.ts

```diff
@@ -111,11 +111,17 @@ async function handleObjectPropertiesDialogCommand(context: azdata.ObjectExplore
 		return;
 	}
 	try {
+		const connectedToLocalhost = context.connectionProfile.serverName === 'localhost';
```

Comment:
This seems pretty hacky - why do we need the "actual" servername? IIRC the URN doesn't even need a name for the server (since you give it a connection to the server already) - so you can just build it up as `Server/Database[@Name='...']` or whatever

Reply:
I tried doing something similar and got and error. Let me try again just in case.

Reply:
If not please work with @corivera and @shueybubbles to try and figure out what's going wrong there - we should at least identify what the actual bug is so we can try to address it quickly.

Reply:
Using `Server/Database[@Name='...']` worked! thank you!